realsense2_camera/test/live_camera/rosci.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2024 Intel Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys, os, subprocess, re, getopt, time
+
+start_time = time.time()
+running_on_ci = False
+if 'WORKSPACE' in os.environ:
+    #Path for ROS-CI on Jenkins
+    ws_rosci = os.environ['WORKSPACE']
+    sys.path.append( os.path.join( ws_rosci, 'lrs/unit-tests/py' ))
+    running_on_ci = True
+else:
+    #For running this script locally
+    #Extract the root where both realsense-ros and librealsense are cloned
+    ws_local = '/'.join(os.path.abspath( __file__ ).split( os.path.sep )[0:-5])
+    #expected to have 'librealsense' repo in parallel to 'realsense-ros'
+    assert os.path.exists( os.path.join(ws_local, 'librealsense')), f" 'librealsense' doesn't exist at {ws_local} "
+    sys.path.append( os.path.join( ws_local, 'librealsense/unit-tests/py' ))
+
+#logs are stored @ ./realsense2_camera/test/logs
+logdir = os.path.join( '/'.join(os.path.abspath( __file__ ).split( os.path.sep )[0:-2]), 'logs')
+dir_live_tests = os.path.dirname(__file__)
+
+from rspy import log, file
+regex = None
+hub_reset = False
+handle = None
+test_ran = False
+device_set = list()
+
+def usage():
+    ourname = os.path.basename( sys.argv[0] )
+    print( 'Syntax: ' + ourname + ' [options] ' )
+    print( 'Options:' )
+    print( '        -h, --help      Usage help' )
+    print( '        -r, --regex     Run all tests whose name matches the following regular expression' )
+    print( '                        ex: --regex test_camera_imu; -r d415_basic')
```

Comment:
I would prefer "example:" or "e.g.,:", not "ex:"

Reply:
fixed